Proposals/ParametricandOpticalAxes/ProposalSummary.md

```diff
@@ -0,0 +1,373 @@
+
+# Proposal: Parametric and Optical Axes
+
+## Administrative Information
+**Proposers name:** Sam Berlow
+
+**Vendor affiliation:** Type Network
+
+**Proposal name:** Parametric Axes
+
+**Date of submission:** 12/6/2017
+
+**New or revised proposal:** New
+
+**Previous revision date:** N/A
+
+## General Technical Information
+
+**Related axes:** See individual axis proposals
+
+**Similar axes:** See individual axis proposals
+
+**Axis type:** Parametric
+
+## Proposed Axis: `xtra`
+
+**Tag:** xtra
+
+**Name:** x transparent
+
+**Axis type:** Optical
+
+**Description:** assigns a “white” per mille value to each instance of the design space
+
+**Valid numeric range:**  -1000 to 2000
+
+**Scale interpretation:** Values should be interpreted as per-mille-of-em
+
+**Recommended or required “Regular” value:** N/A
+
+**Suggested programmatic interactions:** Example: by setting a maximum word space parameter, and adjusting the XTRA units to a min and max. A developer could adjust justified text without tracking letters. Maintaining the apparent weight and width. 
+
+**UI recommendations:** Users may choose to program a variant in connection to direct or conjunctive input for a page description language, or via a user interface
+
+**Script or language considerations:** Many languages and scripts 
+
+**Related axes:** wght, width, opsz
```

Comment:
See the guidance for this field: 

"If this axis is intended to be used specifically in combination with certain other already-registered or proposed axes, identify the axes or proposals."

In this case, I think your intent is that xtra would be used in combination with xopq and other of the axes being proposed. But perhaps you also intend to suggest some kind of interaction with wght, wdth, opsz. It would probably be helpful to elaborate a little on these relationships and interactions. (That wasn't specified in the template, but it will be useful to understand the intent of the axis.) You can give such elaboration in each ProposalSummary for the axis/axes in that particular page, or in an overview page covering all the axes, or some of both: general remarks in the overview, and specific details in the proposal summary.